src/ai/backend/common/web/session/__init__.py

```diff
@@ -146,6 +158,20 @@ def __delitem__(self, key: str) -> None:
 STORAGE_KEY = "aiohttp_session_storage"
 
 
+async def update_expires(request: web.Request) -> str:
+    storage = request.get(STORAGE_KEY)
+    if storage is None:
+        raise RuntimeError("Install aiohttp_session middleware in your aiohttp.web.Application")
+    session: Session = await get_session(request)
+    await storage.update_expires(session)
+    if session is not None:
+        expires_timestamp = int(storage.expires) if storage.expires is not None else 0
+        session.expires = expires_timestamp
+        return datetime.fromtimestamp(timestamp=expires_timestamp, tz=timezone.utc).isoformat()
+    else:
+        raise RuntimeError("Session is None")
```

Comment:
`get_session()` ensures that returned value is not null.
is there any possibility of the returned `session` is `None`?

Reply:
I added conditional statement in order to pass the mypy checks, but I'll fix it.